marker_ids: [1, 7, 5, 2, 3, 4, 6, 8, 9, 10]
min_marker_detection: 2
rate_filt_velocity: 5
rate_filt_time: 0.1
cam2imuTf: "/parameters/cam2imu_blackfly1.yaml"
camera_frame: "camera"
optitrack_data_vector_length: 40
marker_transform_sample_num: 5
filter_const: 0.2
use_optitrack_align: true